portal/apps/org.wso2.carbon.iam.user-portal.app/pom.xml

```diff
@@ -59,6 +59,30 @@
             <groupId>org.wso2.carbon.identity.mgt</groupId>
             <artifactId>org.wso2.carbon.identity.recovery</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-api</artifactId>
+            <version>2.40.0</version>
```

Comment:
Can't we add the versions as properties in parent pom. Add the dependency to DependencyManagement section in the parent.